[2024-11-02 09:08:53,489][__main__][INFO] - Instantiating datamodule <src.datamodules.dogbreed_datamodule.DogBreedImageDataModule>
[2024-11-02 09:08:59,419][__main__][INFO] - Instantiating model <src.models.dogbreed_classifer.TimmClassifier>
[2024-11-02 09:09:01,154][timm.models._builder][INFO] - Loading pretrained weights from Hugging Face hub (timm/resnet18.a1_in1k)
[2024-11-02 09:09:01,875][timm.models._hub][INFO] - [timm/resnet18.a1_in1k] Safe alternative available for 'pytorch_model.bin' (as 'model.safetensors'). Loading weights using safetensors.
[2024-11-02 09:09:01,889][timm.models._builder][INFO] - Missing keys (fc.weight, fc.bias) discovered while loading pretrained weights. This is expected if model is being adapted.
[2024-11-02 09:09:04,877][__main__][INFO] - Instantiating callback <src.train.CustomModelCheckpiont>
[2024-11-02 09:09:09,539][__main__][INFO] - Instantiating callback <lightning.pytorch.callbacks.EarlyStopping>
[2024-11-02 09:09:13,588][__main__][INFO] - Instantiating callback <lightning.pytorch.callbacks.RichModelSummary>
[2024-11-02 09:09:13,589][__main__][INFO] - Instantiating callback <lightning.pytorch.callbacks.RichProgressBar>
[2024-11-02 09:09:21,841][__main__][INFO] - instantiating logger <lightning.pytorch.loggers.CometLogger>
[2024-11-02 09:09:26,716][__main__][INFO] - instantiating logger <lightning.pytorch.loggers.csv_logs.CSVLogger>
[2024-11-02 09:09:29,720][__main__][INFO] - instantiating logger <lightning.pytorch.loggers.MLFlowLogger>
[2024-11-02 09:09:34,829][__main__][INFO] - Instantiating trainer <lightning.Trainer>
[2024-11-02 09:09:59,665][__main__][INFO] - Starting training!
[2024-11-02 09:10:13,092][urllib3.connectionpool][WARNING] - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='www.comet.com', port=443): Read timed out. (read timeout=10)")': /api/rest/v2/write/experiment/file-path
